libs/ledger-live-common/src/modularDrawer/hooks/useRightBalanceNetwork.tsx

```diff
@@ -36,9 +33,7 @@ export function createUseRightBalanceNetwork({ useBalanceDeps, balanceItem }: Ne
       };
     });
 
-    networksWithBalance.sort((a, b) =>
-      compareByBalanceThenFiat(a.balanceData, b.balanceData, discreet),
-    );
+    networksWithBalance.sort((a, b) => compareByBalanceThenFiat(a.balanceData, b.balanceData));
```

Comment:
I think there's a bug here which is already present in develop (so maybe it makes sense as a separate ticket), but the ordering looks incorrect. I think the sort here means the balance components get rearranged, and then mismatched against corresponding network:

<img width="532" height="303" alt="Image" src="https://github.com/user-attachments/assets/625cdb4f-9706-4f24-94b2-41ba20dad0aa" />

<img width="518" height="343" alt="Image" src="https://github.com/user-attachments/assets/6e873e93-ee08-43a3-b56f-68cdefae1047" />

<img width="521" height="349" alt="Image" src="https://github.com/user-attachments/assets/e497713d-1383-4451-ac11-fd64321f2249" />
 
Instead `createUseRightBalanceNetwork` should be returning the `balanceData` for `createNetworkConfigurationHook` to sort

Reply:
i think it makes more sense to fix this in a separate ticket though so i'll create a ticket

Reply:
https://ledgerhq.atlassian.net/browse/LIVE-21901